files/data.yml

```diff
@@ -902,19 +414,12 @@ data:
     CA_DN: /C=GR/O=HellasGrid/OU=Certification Authorities/CN=HellasGrid CA 2016
     hostname: voms2.hellasgrid.gr
     port: '15020'
-- name: rdfa
-  voms:
-  - DN: /DC=org/DC=terena/DC=tcs/C=IT/ST=Roma/O=Istituto Nazionale di Fisica Nucleare/CN=voms.cnaf.infn.it
-    CA_DN: /C=NL/O=GEANT Vereniging/CN=GEANT eScience SSL CA 4
-    hostname: voms.cnaf.infn.it
-    port: '15014'
 - name: vo.darkside.org
   voms:
-  - DN: /DC=org/DC=terena/DC=tcs/C=IT/ST=Roma/O=Istituto Nazionale di Fisica Nucleare
-      - INFN/CN=vomsmania.cnaf.infn.it
-    CA_DN: /C=NL/O=GEANT Vereniging/CN=GEANT eScience SSL CA 4
-    hostname: vomsmania.cnaf.infn.it
-    port: '15008'
+  - DN: /O=Acme Co/CN=Kubernetes Ingress Controller Fake Certificate
```

Comment:
😮

Reply:
Good catch, it needs to be investigated

Reply:
So the ID card is at https://operations-portal.egi.eu/vo/view/voname/vo.darkside.org#VOV_section.

It's not clear to me if the URL is deprecated or wrong (it's even using an http URL)
This will have to be reviewed with the community contacts.

Reply:
I've manually removed that VO from the list.